chepstow/agent.py

```diff
@@ -11,6 +11,7 @@ class Agent(object):
     """
 
     def __init__(self):
+        print(self.clay)
```

Comment:
This is a good idea, but looks like you forgot to define the variable.

Reply:
Oh yea? Says you.